secio/src/codec/secure_stream.rs

```diff
@@ -358,10 +358,12 @@ mod tests {
                     Vec::new(),
                 );
                 let mut handle = secure.create_handle().unwrap();
+                tokio::spawn(secure.for_each(|_| Ok(())).map_err(|_| ()));
+
                 let _ = handle.write_all(&nonce);
                 let _ = handle.write_all(&data_clone[..]);
-
-                tokio::spawn(secure.for_each(|_| Ok(())).map_err(|_| ()));
+                // wait test finish, don't drop handle
+                thread::sleep(time::Duration::from_secs(10));
```

Comment:
use `channel` to synchronous

Reply:
It's not a problem with the channel.

Because the handle is dropped, the stream can't get the data from the dropped stream.